seeder/src/renderer/main.css

```diff
@@ -25,7 +25,5 @@
 }
 
 * {
-	outline: none !important;
-	box-sizing: border-box;
-  user-select: none;
+  @apply !outline-none select-none;
```

Comment:
I am still not a fan of this method. Can you do it in components selectively?

Reply:
It is a pain you know right?
what feels off?

Reply:
>  select-none;

Reply:
https://github.com/coreproject-moe/CoreProject/pull/3929/commits/d4c4c9fc3bee6bf59108af9360572e15d72b7eb5: `select-none` can be added to body el, and for elements you want to select: `select-text/auto` can be added